pkg/services/authz/zanzana/zanzana.go

```diff
@@ -174,3 +176,19 @@ func TranslateToCheckRequest(namespace, action, kind, folder, name string) (*aut
 
 	return req, true
 }
+
+type ResourceInfo struct {
+	Typ      string
+	Group    string
+	Resource string
+}
```

Comment:
Do we need the type filed? 

Also when building this we can also set `GroupResource` property

```go
func TranslateResource(kind string) (ResourceInfo, bool) {
	translation, ok := resourceTranslations[kind]
	info := ResourceInfo{
		Typ: translation.typ,
		Group: translation.group,
		Resource: translation.resource,
		GroupResource: common.FormatGroupResource(translation.Group, translation.Resource)
	}
	return info, ok
}
```
Then we don't have to format it later

Reply:
Hm, that makes sense